internal/provider/provider.go

```diff
@@ -127,6 +127,9 @@ func New() *schema.Provider {
 			"boundary_user":                                     resourceUser(),
 			"boundary_worker":                                   resourceWorker(),
 		},
+		DataSourcesMap: map[string]*schema.Resource{
+			"boundary_user": dataSourceUser(),
```

Comment:
You will want to merge with main as this will conflict now that scope datasources have merged